ssz/simple-serialize.md

```diff
@@ -313,12 +329,16 @@ recursively:
   bitvector.
 - `mix_in_length(merkleize(pack(value), limit=chunk_count(type)), len(value))`
   if `value` is a list of basic objects.
+- `mix_in_length(merkleize_progressive(pack(value)), len(value))` if `value` is
```

Comment:
looks like here and below, merkleize_progressive should be called without the second arg (or with second arg == 1)

Reply:
? There is only a single arg (meaning it uses the default value of 1 for `num_leaves`)

- `mix_in_length`
  - arg 1: `merkleize_progressive(pack(value))` (meaning default value `num_leaves=1`)
  - arg 2: `len(value)`

Reply:
whoops, miscounted parens. Looks good.